draft-ietf-teep-architecture.md

```diff
@@ -459,26 +450,77 @@ all components are further explained in the following paragraphs.
 
   - Certification Authority (CA):  Certificate-based credentials used for
     authenticating a device, a TAM and an SP.  A device embeds a list
-    of root certificates (trust anchors), from trusted CAs that a TAM
+    of root certificates (Trust Anchors), from trusted CAs that a TAM
     will be validated against.  A TAM will remotely attest a device
     by checking whether a device comes with a certificate from a CA
     that the TAM trusts.  The CAs do not need to be the same;
     different CAs can be chosen by each TAM, and different device CAs
     can be used by different device manufacturers.
 
 ## Different Renditions of TEEP Architecture
+There is nothing prohibiting a device from implementing multiple TEEs. In
+addition, some TEEs present themselves as separate containers within memory 
+without a controlling manager within the TEE - for example SGX. In these cases, 
```

Comment:
suggest moving "for example SGX" up to in front of "some TEEs" in line 462, as in
"some TEEs (for example, SGX) present themselves as separate containers..."

Reply:
Good suggestions.